libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -147,6 +147,15 @@ const AP_Param::GroupInfo AC_AttitudeControl::var_info[] = {
     AP_GROUPEND
 };
 
+// get the yaw rate limit
```

Comment:
maybe this comment should specify the units

Reply:
done, both the comment and function name